demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureTypeService.php

```diff
@@ -334,8 +333,9 @@ public function calculateStatementFieldDefinitionChanges(
     }
 
     /**
-     * Adds the fieldDefinitions and participationGuestOnly (it is never send in request since it can't be manually changed) of the original procedure type to the request. Also adds the original ID for easier
-     * error handling in case of a redirect to the form.
+     * Adds the fieldDefinitions and participationGuestOnly (it is never send in request since it can't be
+     * manually changed) of the original procedure type to the request. Also adds the original ID for easier error
+     * handling in case of a redirect to the form.
      */
     public function addMissingRequestData(
         string $formName,
```

Comment:
$formName is not used anymore. When this is correct you should delete it

Reply:
To keep this PR small I've shift the cleanup to #2292